addons/web/static/tests/legacy/helpers/test_utils_mock.js

```diff
@@ -317,9 +319,9 @@ function intercept(widget, eventName, fn, propagate) {
  * @param {function} [params.mockRPC]
  * @param {number} [params.fieldDebounce=0] the value of the DEBOUNCE attribute
  *   of fields
- * @param {boolean} [params.debounce=true] if false, patch _.debounce to remove
+ * @param {boolean} [params.debounce=true] if false, patch _debounce to remove
```

Comment:
I checked the whole codebase and this param seems never used.
You can remove it.